lib/matplotlib/_mathtext.py

```diff
@@ -2050,10 +2060,27 @@ def accent(self, s, loc, toks):
             accent_box = AutoWidthChar(
                 '\\' + accent, sym.width, state, char_class=Accent)
         else:
+            # Check if accent and character can be combined
```

Comment:
One can possibly consider splitting the accents into those that may have precomposed characters and those that may not.
https://en.wikipedia.org/wiki/List_of_precomposed_Latin_characters_in_Unicode

Possibly one should check that the character is one of the standard latin characters as well, although that may lead to that those precomposed with two accents may not work (which should be checked if they even do to start with...).